src/test/java/org/onecx/search/config/rs/external/v1/SearchConfigControllerV1Test.java

```diff
@@ -0,0 +1,393 @@
+package org.onecx.search.config.rs.external.v1;
+
+import static io.restassured.RestAssured.given;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+import jakarta.ws.rs.core.MediaType;
+
+import org.junit.jupiter.api.Test;
+import org.onecx.search.config.rs.test.AbstractTest;
+import org.tkit.quarkus.test.WithDBData;
+
+import gen.io.github.onecx.search.config.v1.model.*;
+import io.quarkus.test.junit.QuarkusTest;
+
+@QuarkusTest
```

Comment:
Check the test in onecx-permission-svc and use @TestHTTPEndpoint(...RestController.class)